docs/install.rst

```diff
@@ -3,6 +3,11 @@ How to install
 
 .. toctree::
 
+Requirement
+-----------
+
+`trepan3k` works with python >=3.7 and <3.9.
```

Comment:
Um. It does work on 3.10 and probably 3.11. You just don't get the decompilation aspect.

Reply:
Actually, it works on Python versions _before_ 3.7, down to 3.2, as well. See https://pypi.org/project/trepan3k/#files

or the left-hand side of https://pypi.org/project/trepan3k/ under "Programming Language"

Where did you come up with this information?

Reply:
> > Where did you come up with this information?
> 
> Sorry, I have misunderstand something.
> 
> So if I understand well, trepan3k can work for python >=3.2.

Yes. Develop the debugger for versions 3.2-3.5 is done in the branch named `python-3.2`.

> About the decompilation feature, it supports decompilation for python >=3.2 and <3.7, it doesn't support decompilation for python > 3.9. And for python 3.7 and 3.8, did it support decompilation?

You decompilation is supported up to and including 3.8. How good often or well this works is based of course on the goodness of the decompiler.